data/events.js

```diff
@@ -1203,14 +1203,14 @@ compiling it at delivery).`
 		},
 		inherit: 'pre_login_client',
 		fields: {
-			reason: `Short reason; see the short to long reason mapping in the table below.`,
+			reason: `Short reason; see the short reason to description mapping in the table above.`,
```

Comment:
could we use [[link,table_link]] here?

Reply:
no idea, @slusarz ?

Reply:
you can just use [[link, name of link]]

Reply:
But this is all generated stuff. I'd need to link to "Field list", which is also generated. I don't know if I can somehow place an anchor there.

Reply:
I'm guessing it would require changing the javascript code to generate anchors to all of them based on some template name.

Reply:
Commonmark allows HTML, so no need to over complicate.  Just use HTML anchor linking.

i.e.:
```
reason: `Short reason; see the short reason to description mapping in the <a href="#pre_login_client_error_codes">table above</a>.`,
```

and

```
<span id="pre_login_client_error_codes">Error Codes:</span>
```